.travis.yml

```diff
@@ -15,12 +17,17 @@ before_install:
   # The next couple lines fix a crash with multiprocessing on Travis and are not specific to using Miniconda
   - sudo rm -rf /dev/shm
   - sudo ln -s /run/shm /dev/shm
+  - git clone https://github.com/anderspitman/tidepool-docker
+  - tidepool-docker/build_containers tidepool-docker anderspitman
```

Comment:
Definitely. It's slightly hacked together at the moment, but with a little cleanup I think it would be generally useful.